cypress/support/config/settings.js

```diff
@@ -4030,11 +4030,17 @@ module.exports = () => ({
             enabled: false,
           },
           test: {
-            paths: ['/mundo/articles/ce42wzqr2mko'],
+            paths: [
+              '/mundo/articles/ce42wzqr2mko',
+              '/mundo/articles/ce4krqk1334o',
```

Comment:
The cypress tests are currently failing, was there a reason to add these additional pages to the cypress tests?

Reply:
Hmmm... the failure seems to be due to bundle size slightly increasing. I added this page to help us implement and test the anchor links but I doubt that it's part of what gets added to bundle. I'll update the bundle size and add a couple of tests

Reply:
Hey @mukama, no probs adding the page as a test asset, but adding it to the cypress tests will make our test suite take longer. I'd only add it if you think cypress tests specifically will test something relating to anchor links, I don't think that's the case here?

Reply:
@andrewscfc the increase in bundle size seems to be originating from using the URL API. Babel adds an extra 17kb transcribing and polyfilling this API. I've therefore revert back to using `const anchor = locator.split('#')[1];`

Reply:
I could add a Cypress test to test this if we want it.